ansibullbot/triagers/plugins/shipit.py

```diff
@@ -422,25 +423,21 @@ def get_submitter_facts(issuewrapper, meta, module_indexer, component_matcher):
     }
 
     login = issuewrapper.submitter
-    emails = set()
-    for k,v in component_matcher.email_cache.items():
-        if v == login:
-            emails.add(k)
-    for k,v in module_indexer.emails_cache.items():
-        if v == login:
-            emails.add(k)
+    all_emails = itertools.chain(
+        component_matcher.email_cache.items(),
+        module_indexer.emails_cache.items(),
+    )
+
+    emails = set(k for k, v in all_emails if v == login)
 
     email_map = \
         component_matcher.gitrepo.get_commits_by_email(emails)
 
-    for k,v in email_map.items():
-        sfacts['submitter_previous_commits'] += v['commit_count']
+    for value in list(email_map.values()):
```

Comment:
Just curious as to why casting to list explicitly? I know it's because of some changes in Python 3 but does it not work without it?

Reply:
Look good otherwise :+1:

Reply:
Yes. It would work without casting. This syntax will work for Python 2 and 3.